src/libs/SidebarUtils.ts

```diff
@@ -126,11 +139,13 @@ function getOrderedReportIDs(
         // However, this code needs to be very performant to handle thousands of reports, so in the interest of speed, we're just going to disable this lint rule and add
         // the reportDisplayName property to the report object directly.
         // eslint-disable-next-line no-param-reassign
-        report.displayName = ReportUtils.getReportName(report);
+        report.displayName = filterDisplayName(ReportUtils.getReportName(report));
+
+        const hasRBR = report.reportID in reportIDsWithErrors || reportIDsWithViolations.has(report.reportID);
 
         const isPinned = report.isPinned ?? false;
         const reportAction = ReportActionsUtils.getReportAction(report.parentReportID ?? '', report.parentReportActionID ?? '');
-        if (isPinned || ReportUtils.requiresAttentionFromCurrentUser(report, reportAction)) {
+        if (isPinned || hasRBR || ReportUtils.requiresAttentionFromCurrentUser(report, reportAction)) {
             pinnedAndGBRReports.push(report);
```

Comment:
```suggestion
            pinnedAndBrickRoadReports.push(report);
```

I think this includes RBR too, let's rename so it's more generic. And also update comment on line [120](https://github.com/Expensify/App/blob/1eca3e126b89bf7dd762447e26951654611bdbfa/src/libs/SidebarUtils.ts#L120) to say:

>// 1. Pinned/GBR/RBR - Always sorted by reportDisplayName